src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -43,9 +43,10 @@ def __init__(self):
     for extension in extensions_to_strip:
       app_name = utils.strip_from_right(app_name, extension)
 
-    self.trials = {}
     for trial in data_types.Trial.query(data_types.Trial.app_name == app_name):
-      self.trials[trial.app_args] = trial.probability
+      self.trials[trial.app_args] = {'probability': trial.probability}
```

Comment:
Can you created a namedtuple or class for this containing the two properties (probability and contradicts) ?

Reply:
done, created class called AppArgs